android/src/main/java/com/magicleap/magicscript/scene/nodes/dropdown/DropdownItemsListNode.kt

```diff
@@ -0,0 +1,159 @@
+package com.magicleap.magicscript.scene.nodes.dropdown
+
+import android.content.Context
+import android.view.LayoutInflater
+import android.view.View
+import android.widget.ListView
+import com.facebook.react.bridge.ReadableMap
+import com.google.ar.sceneform.rendering.Renderable
+import com.magicleap.magicscript.R
+import com.magicleap.magicscript.ar.ViewRenderableLoader
+import com.magicleap.magicscript.scene.nodes.base.TransformNode
+import com.magicleap.magicscript.scene.nodes.base.UiNode
+import com.magicleap.magicscript.utils.Utils
+import com.magicleap.magicscript.utils.Vector2
+import com.magicleap.magicscript.utils.getSizeInMeters
+
+class DropdownItemsListNode(
+    initProps: ReadableMap,
+    context: Context,
+    viewRenderableLoader: ViewRenderableLoader
+) : UiNode(initProps, context, viewRenderableLoader) {
+
+    companion object {
+        const val DEFAULT_ITEMS_TEXT_SIZE = 0.025F
+
+        const val RENDER_PRIORITY = 5 // 4 is default, 5 means we draw at top of other views
```

Comment:
We should create RenderPriority object or something because it is not clear right now.